src/app/public/modules/summary-action-bar/summary-action-bar-adapter.service.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Injectable,
+  Renderer2,
+  RendererFactory2,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import {
+  Subscription
+} from 'rxjs/Subscription';
+
+import 'rxjs/add/observable/fromEvent';
+
+import {
+  SkyWindowRefService
+} from '@skyux/core';
+
+import {
+  SkySummaryActionBarType
+} from './types';
+
+@Injectable()
+export class SkySummaryActionBarAdapterService implements OnDestroy {
+
+  private renderer: Renderer2;
+
+  private resizeSubscription: Subscription;
+
+  constructor(
+    private rendererFactory: RendererFactory2,
+    private windowRef: SkyWindowRefService
+  ) {
+    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
+  }
+
+  public ngOnDestroy() {
+    if (this.resizeSubscription) {
+      this.resizeSubscription.unsubscribe();
+    }
+  }
+
+  public adjustWindowMarginForActionBar(destroying?: boolean): void {
+    const window = this.windowRef.getWindow();
+    const body = window.document.body;
+    if (destroying) {
+      this.renderer.setStyle(body, 'margin-bottom', '');
```

Comment:
Could this be `this.renderer.removeStyle(body, 'margin-bottom')`, so that it is not destructive of any external CSS that might be applied to the `body`?

Reply:
No. `renderer.removeStyle` requires all three values. I matched what the original component did here but am open if you have a better suggestion. I agree that this feels a little dangerous. I'll research a better approach.

Reply:
Changed. 

(Please ignore any previous comment here that you might have seen - I misread the API documentation and realized it only after I had commented.)